### PR TITLE
Add install scripts for Minikube Kubernetes without VM

### PR DESCRIPTION
**Pull request by chc5**
The installation scripts for minikube works on Ubuntu 18.04

### CODE REVIEW

File: scripts/check [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Kubernetes Dependencies
+command -v kubectl >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency kubectl does not exist"
+}
+command -v minikube >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency minikube does not exist"
+}
+command -v docker >/dev/null 2>&1 || {
+  echo >&2 "check.sh: The dependency docker does not exist"
+}
```
Comment (yliu0715) [inline scripts/check]: Just a suggestion but can't you turn this into a function like 
`does_cmd_exist() {
  command -v $1 > /dev/null 2>&1
}` and then call it for each dependency that we need to install by calling `does_cmd_exist kubectl` or something

Comment (chc5) [reply scripts/check]: Created does_cmd_exist

File: scripts/check [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+does_cmd_exist_for_service() {
+  command -v ${arg} >/dev/null 2>&1 || {
+    echo >&2 "check: The dependency $2 for the service $1 does not exist"
+  }
+}
+
+# Checks if the minimal Kubernetes dependencies exist
+does_cmd_exist kubernetes kubectl
+does_cmd_exist kubernetes minikube
+does_cmd_exist kubernetes docker
```
Comment (yliu0715) [inline scripts/check]: Your function name is not matching lol. `does_cmd_exist` and `does_cmd_exist_for_service`

Comment (yliu0715) [reply scripts/check]: And I think you meant to write `$2` instead of `${arg}` on line 14?

Comment (chc5) [reply scripts/check]: Fixed

**Pull request merged**